lib/model.js

```diff
@@ -2265,7 +2265,7 @@ class Model {
    * {@link Model.findAll} for a full specification of find and options
    *
    * @param {Object}      options find and create options
-   * @param {Object}      options.where where A hash of search attributes.
+   * @param {Object}      options.where where A hash of search attributes. Will be appended to defaults if create runs. 
```

Comment:
If `where` is a plain object it will be appended with `defaults` to create a new instance